owsi-core/owsi-core-components/owsi-core-component-wicket-more/src/main/java/fr/openwide/core/wicket/more/markup/html/collection/GenericEntityListView.java

```diff
@@ -8,10 +8,20 @@
 import org.apache.wicket.model.IModel;
 
 import fr.openwide.core.jpa.business.generic.model.GenericEntity;
+import fr.openwide.core.wicket.more.markup.repeater.collection.CollectionView;
 import fr.openwide.core.wicket.more.model.GenericEntityModel;
+import fr.openwide.core.wicket.more.model.ReadOnlyCollectionModel;
 
 /**
  * A {@link ListView} akin to {@link AbstractGenericEntityCollectionView}, which is itself a {@link RefreshingView}.
+ * @deprecated use {@link CollectionView} with {@link ReadOnlyCollectionModel} and {@link GenericEntityModel} instead : 
+ * <pre>
+ * {@code 
+ * GenericEntity*View<T>("id", model);
+ * -> 
+ * new CollectionView<T>("id", ReadOnlyCollectionModel.of(model, GenericEntityModel.<T>factory()));
```

Comment:
Don't forget to replace the sentence just above, too, in order to remove the reference to `ReadOnlyCollectionModel`